packages/bot/src/stores/journal-store.js

```diff
@@ -13,7 +17,16 @@ export default class JournalStore {
         return this.root_store.core.common.server_time;
     }
 
+    filter_list = [
+        { id: message_types.ERROR, label: 'Error messages', onChange: this.filterMessage },
+        { id: message_types.NOTIFY, label: 'Notifications', onChange: this.filterMessage },
+        { id: message_types.SUCCESS, label: 'System log', onChange: this.filterMessage },
+    ];
+
+    unfilter_messages = [];
+
     @observable messages = [];
+    @observable checked_filter = getSetting('journal_filter') || this.filter_list.map(filter => filter.id);
```

Comment:
Could you use plurals for variables that can contain multiples of something? i.e. `checked_filters`